CONTRIBUTING.md

```diff
@@ -1,11 +1,54 @@
+## Minor Changes
 
-Once the design has been finalized in the Google Doc, support for the change must be implemented in the service controller of a platform easily accessible to users. Currently Cloud Foundry is the most appropriate platform for this, but we look forward to other platforms adding support for the broker API soon, and one of those platforms could take the initiative of implementing support for the feature earlier. 
+Minor change proposals to the specification, changes such as editorial bugs
+or enhancements that do not modify the semantics of the specification or
+syntax of the API, can be suggsted via a Github Issue or Pull Request (PR).
 
-Once support for the change has been implemented, a PR should be to this repo for the change to the broker API specificaion. By this point, the API interactions should be well understood and there should be no technical surprises; we expect the only discussion necessary on PRs to be for wordsmithing and formatting.
+If there may be a need for some discussion around how best to
+address the concern then opening an Issue, prior to doing the work to develop
+a PR, would be best.  These minor issues do not need a formal review, per the
+[process](#prissue-review-process) described below, rather the issue
+should be used to come to a conscensus around a PR that will eventually be
+submitted.
 
-All contributors sending Pull Requests (PRs) must have a Contributor License Agreement 
-on file as either an [individual](https://www.cloudfoundry.org/pdfs/CFF_Individual_CLA.pdf) 
+If the proposed change is non-controversial (e.g. a typo) then a PR can be
+submitted directly without opening an issue.
+
+Either way, once a PR is submitted it will be reviewed per the
+[process](#prissue-review-process) described below.
+
+## Design Change Proposals
+
+New design proposals to the API spec should be submitted by opening a Github issue with a link to a Google Doc containing the proposal. Proposals should contain motivation, use cases, and detailed example of proposed changes to API interactions. Collaboration on the design, fleshing out of use cases, etc can occur as comment discussions in the Google Doc, as well as on our weekly calls.
+
+Once the design has been finalized in the Google Doc, support for the change must be implemented in the service controller of a platform easily accessible to users. Currently Cloud Foundry is the most appropriate platform for this, but we look forward to other platforms adding support for the broker API soon, and one of those platforms could take the initiative of implementing support for the feature earlier.
```

Comment:
> support for the change must be implemented in the service controller of a platform easily accessible to users.

The service broker API aims at supporting use-cases of multiple actors:
- platforms (CF, K8S and others), including:
   - Service controllers  
   - Platform UIs and tooling
- service authors (developing service brokers to offer services)
- potentially intermediates (e.g. marketplace providers)

We probably want to avoid a change that won't support some of these actors and would remain non-adopted by the community. While the design discussions should alleviate most obvious high level objections from these actors, it seems key to have running code validating most important use-cases supported by the change.

Suggesting the following alternative wording:

```
The finalized design needs to be validated with minimal implementation covering most important targeted use-cases. This may vary from:
- platforms support such as :
   - Service controllers supporting the API change
   - Platform UIs and tooling supporting or leveraging the change
- service broker implementations leveraging the change (see [example route services](http://docs.cloudfoundry.org/services/route-services.html#examples) as an example)
```